package.json

```diff
@@ -13,7 +13,7 @@
     "changelog": "npm run changelog:generate && npm run changelog:add",
     "changelog:add": "git add CHANGELOG.md",
     "changelog:generate": "github_changelog_generator --future-release $npm_package_version",
-    "coverage": "npm test && nyc report --reporter=lcov > coverage/coverage.lcov && codecov",
+    "coverage": "nyc jest",
```

Comment:
We are not using `jest` here `mocha` instead

Reply:
oh yep, maybe we should migrate..